valgrind.suppressions

```diff
@@ -499,3 +499,18 @@
    fun:btrfs_reflink_with_progress
    ...
 }
+{
+   _g_daemon_vfs_init
```

Comment:
do we really need this VFS overhead? Can you not create a GIOStream based on the filename alone without GFile?

Reply:
I've switched to g_task_run_in_thread() now.